cli/Cargo.toml

```diff
@@ -30,8 +30,8 @@ winapi = "0.3.9"
 
 [dependencies]
 deno_core = { path = "../core", version = "0.63.0" }
-deno_doc = "0.1.11"
-deno_lint = { version = "0.2.3", features = ["json"] }
```

Comment:
Nice to get rid of the `json` feature :+1:

Reply:
Yeah, it was superfluous